CHANGELOG.md

```diff
@@ -14,6 +14,10 @@
 - [v0.1.0-rc2](#v010-rc2)
 - [v0.1.0-rc1](#v010-rc1)
 
+## Unreleased
+
+Changes that are in `main` but have not yet made it to a release.
```

Comment:
Maybe add some kind of note here to indicate that unreleased changes may be rolled back or changed prior to a release.

Reply:
```suggestion
Changes that are in `main` but have not yet made it to a release.

**NOTE**: unreleased changes are subject to rollback or change prior to landing in a release
```

Reply:
Sounds good. Change in 1e06e53e677c12d76f8fe9ec6570ffea61908737